tools/Test/SmokeTest/RmCoreSmokeTests.ps1

```diff
@@ -171,11 +172,20 @@ $startTime = Get-Date
 $resourceCommands | ForEach-Object {
     $testName = $_.Name
     $script = $_.Command
+    $retry = if($null -eq $_.retry) {0} Else {$_.retry}
+    $sleep = if($null -eq $_.sleep) {30} Else {$_.sleep}
+    if($null -ne $_.Since -and "Core" -eq $PSVersionTable.PSEdition) {
+        if($PSVersionTable.PSVersion -lt [System.Version]$_.Since) {
+            Write-Output "Skip test $testName"
+            $testInfo.SkippedCount += 1
+            return
+        }
+    }
     Write-Output "Running test $testName"
     $testStart = Get-Date
     try
     {
-         &$script
+        Retry-AzCommand -Name $testName -Command $script -Retry $retry -Sleep $sleep
          $testInfo.PassedCount += 1
```

Comment:
```suggestion
         Retry-AzCommand -Name $testName -Command $script -Retry $retry -Sleep $sleep
         $testInfo.PassedCount += 1
```
Add a space to keep same indent.

Reply:
Updated the code. The problem indeed is indent of other lines is incorrect.